### DISCUSSION TITLE
Core implementation in NetBeans

### DISCUSSION THREAD

**Discussion by Chris2011** [moacirrf/netbeans-markdown#45] [Show and tell]
Hey, have you seen this? https://github.com/apache/netbeans/pull/5891. Interesting that it was somehow easy :)

**Comment by moacirrf** [comment]
Humm interesting, a core implementation will be pretty usefull, it will be a lot more good if they provide it as a service...Java doc will work with markdown

**Comment by Chris2011** [comment]
So we can try to improve this for example. I dunno how feature rich it is but it is a beginning :)

**Reply by Chris2011** [reply]
Just to let you know
